src/main/resources/format-pages.txt

```diff
@@ -857,7 +857,7 @@ reader = HitachiReader
 
 [I2I]
 extensions = .i2i
-developer = `Biomedical Imaging Group, UMass Medical School <http://invitro.umassmed.edu/>`_
+developer = `Biomedical Imaging Group, UMass Medical School <http://umassmed.edu/>`_
```

Comment:
The Biomedical Imaging Group is still at http://invitro.umassmed.edu/ so necessary to make the URL less specific?

Reply:
Can you access it? That link had been dead and is still timing out for me.

Reply:
I can. It's slow but it comes up. I'll drop them a line to warn them.